apps/emqx/src/emqx_ds_schema.erl

```diff
@@ -145,26 +158,26 @@ fields(builtin_raft) ->
                     importance => ?IMPORTANCE_HIDDEN
                 }
             )},
-        %% TODO: Deprecate once cluster management and rebalancing is implemented.
-        {"n_sites",
+        {replication_factor,
             sc(
                 pos_integer(),
                 #{
-                    default => 1,
-                    importance => ?IMPORTANCE_HIDDEN,
-                    desc => ?DESC(builtin_n_sites)
+                    default => 3,
+                    importance => ?IMPORTANCE_MEDIUM,
+                    desc => ?DESC(builtin_raft_replication_factor)
                 }
             )},
-        {replication_factor,
+        {n_sites,
             sc(
                 pos_integer(),
                 #{
-                    default => 3,
-                    importance => ?IMPORTANCE_HIDDEN
+                    default => 1,
+                    importance => ?IMPORTANCE_LOW,
```

Comment:
Q: why is it necessary to expose those two previously hidden configs (`n_sites` and `replication_factor`) for this change?

Reply:
It's not. I was slightly surprised to find out these are hidden even though we explicitly talk about them in the documentation, so decided to make them visible. Do you think it'd be a better idea to move this change to a separate PR?